apps/pm-user/src/components/molecules/mycard/MyCard.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
Please change name of the molecule folder "mycard" to camel case => "myCard"

Reply:
this issue is resolved in next commit